core/tests/account.go

```diff
@@ -197,6 +197,28 @@ func (baseAccount *BaseAccount) GetConfigForProvider(providerKey schemas.ModelPr
 				BufferSize:  10,
 			},
 		}, nil
+	case schemas.Vertex:
+		return &schemas.ProviderConfig{
+			NetworkConfig: schemas.NetworkConfig{
+				DefaultRequestTimeoutInSeconds: 30,
+				MaxRetries:                     1,
+				RetryBackoffInitial:            100 * time.Millisecond,
+				RetryBackoffMax:                2 * time.Second,
+			},
+			MetaConfig: &meta.VertexMetaConfig{
+				ProjectID: os.Getenv("VERTEX_PROJECT_ID"),
+				ModelRegions: map[string]string{
+					"google/gemini-2.0-flash-001":      "us-central1",
```

Comment:
Thes are not hardcoded regions. So depends on if you are using serverless endpoints or deploying yourselves. So we can keep it as is coming from the user for the request. 

Also one thing we need to test here is - if it comes from google-genai SDK - our flow still holds true or not?

Reply:
sorry didn't get comes from google-genai sdk bit

Reply:
as in you can hit bifrost like

```python
from google import genai

client = genai.Client(api_key=geminiApiKey, base_url="bifrost_url")
```